docs/remote/attach-container.md

```diff
@@ -0,0 +1,79 @@
+g---
+Order: 9
+Area: remote
+TOCTitle: Attach to a Running Container
+PageTitle: Attach to a running container using Visual Studio Code Remote Development
+ContentId: ed14ef07-f44c-4675-b95b-cb5faffc7abb
+MetaDescription: Attach to a running container using Visual Studio Code Remote Development
+DateApproved: 7/9/2020
+---
+# Attach to a running container
+
+While using Visual Studio Code to spin up a new container can be useful in many situations, it may not match your workflow and you may prefer to "attach" VS Code to an already running Docker container - regardless of how it was started. Once attached, you can install extensions, edit, and debug like you can when you open a folder in a container using [devcontainer.json](/docs/remote/devcontainerjson-reference.md).
+
+## Attach to a Docker container
+
+To attach to a Docker container, either select **Remote-Containers: Attach to Running Container...** from the Command Palette (`kbstyle(F1)`) or use the **Remote Explorer** in the Activity Bar and select the **Connect to Container** inline action on the container you want to connect to.
+
+![Containers Explorer screenshot](images/containers/containers-attach.png)
```

Comment:
We could update this image since the Remote Explorer now includes a drop down to choose between WSL, containers, or SSH, and the VS Code debug icon is different as well.